support-frontend/assets/pages/supporter-plus-landing/components/threeTierTsAndCs.tsx

```diff
@@ -38,22 +43,28 @@ const discountSummaryCopy = (currency: string, planCost: TierPlanCosts) => {
 	}
 };
 
-export function ThreeTierDisclaimer({
-	planCost,
+export function ThreeTierTsAndCs({
+	tsAndCsContent,
 	currency,
-}: ThreeTierDisclaimerProps): JSX.Element {
+}: ThreeTierTsAndCsProps): JSX.Element {
+	let counter = 0;
 	return (
 		<>
-			{!!planCost.discount && (
-				<div css={container}>
-					<p>
-						*Digital + Print offer is {discountSummaryCopy(currency, planCost)}{' '}
-						afterwards unless you cancel. Offer only available to new
-						subscribers who do not have an existing subscription with the
-						Guardian.
-					</p>
-				</div>
-			)}
+			{tsAndCsContent.map((tcContent) => {
+				if (tcContent.planCost.discount) {
+					counter++;
```

Comment:
We could call `counter` the same as [this](https://github.com/guardian/support-frontend/pull/5809/files#diff-32a6a1fc8c0ca224a5589f853b87e2d3804385b6c982209375d1a835f056b189R84) - and potentially just leave a comment to that line as they are coupled and it would make it easier to find later.

Reply:
Yep, great idea, I should of renamed this as did the other one, have placed a comment the explain the coupling and where it is.